PowerUp/app/src/main/res/values/dimens.xml

```diff
@@ -51,7 +51,7 @@
     <dimen name="powerup_icon_width">150dp</dimen>
     <dimen name="powerup_width">304dp</dimen>
     <dimen name="power_bar_view_height">75dp</dimen>
-    <dimen name="game_text_size">20dp</dimen>
+    <dimen name="game_text_size">17dp</dimen>
```

Comment:
Why is the text size reduced? This is causing an issue in the phone portrait mode of this screen (which should be refactored). We should not be introducing this here